core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -63,4 +62,24 @@
   simple-editor-sidebar li.sidebar-item-indented {
     margin-left: 25px;
   }
+  simple-editor-sidebar .sidebar {
+    padding-bottom: 80px;
+  }
+  simple-editor-sidebar .full-editor-switch {
+    font-size: 1.0em;
+    background-color: #f3fbf1;
+    bottom: 0;
+    position: fixed;
+    padding-top: 5px;
+  }
+  ::-webkit-scrollbar {
+    width: 8px;
+  }
+  ::-webkit-scrollbar-track {
+    background-color: rgba(0,0,0,.1);
```

Comment:
Is there any way for all these customizations to apply only for the sidebar scrollbar, and not the scrollbar on the right side of the page?